src/index.js

```diff
@@ -0,0 +1,237 @@
+  city: 'Orlando',
+  lat: '',
+  lon: '',
+};
+
+// API calls
+const getLatLon = () => {
+  const inputCityElement = document.getElementById('inputCity').value;
+  axios
+    .get('http://127.0.0.1:5000/location', { params: { q: state.city } })
+    .then((response) => {
+      state.lat = response.data[0].lat;
+      state.lon = response.data[0].lon;
+      console.log(state.lat);
+      console.log(state.lon);
+      getRealTemp();
+    })
+    .catch((error) => {
+      console.log('Error!', error);
+    });
+};
+
+const getRealTemp = () => {
+  console.log(state.lat);
+  console.log(state.lon);
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: state.lat,
+        lon: state.lon,
+      },
+    })
+    .then((response) => {
+      console.log(response);
+      // const tempValueK = response.data.current.temp;
+      const tempValueK = response.data.current.temp;
+      const tempValueF = ((tempValueK - 273.15) * 9) / 5 + 32;
+      const tempF = Math.floor(tempValueF);
+      state.tempF = tempF;
+      displayRealTemp(tempF);
+      landscapeChangeAction();
+      tempValueColorAction();
+    })
+    .catch((error) => {
+      console.log('Error!', error);
+    });
+};
+
+const displayRealTemp = (tempF) => {
+  const tempValueElement = document.getElementById('tempValue');
+  // tempValueElement.textContent = state.tempF;
+  tempValueElement.textContent = tempF;
+};
+
+// Function to increase temp Value by clicking up arrow
+const incTempAction = () => {
+  state.tempValue += 1;
+  const tempDigit = document.getElementById('tempValue');
+  tempDigit.textContent = state.tempValue;
+  tempValueColorAction();
+  landscapeChangeAction();
+};
+
+// Function to decrease temp Value by clicking up arrow
+const decTempAction = () => {
+  state.tempValue -= 1;
+  const tempDigit = document.getElementById('tempValue');
+  tempDigit.textContent = state.tempValue;
+  tempValueColorAction();
+  landscapeChangeAction();
+};
+
+// Function to change color of temp value according to temp range
+const tempValueColorAction = () => {
+  console.log('tempValueColor being called');
+  // document.getElementById('tempValue').style.color = 'orange';
+  // state.color = 'orange';
+  const tempColorElement = document.getElementById('tempValue');
+  // tempColorElement.style.color = 'orange';
+  // tempColorElement.className = color;
+  if (state.tempValue <= 49) {
+    tempColorElement.style.color = 'teal';
+  } else if (50 <= state.tempValue && state.tempValue <= 59) {
+    tempColorElement.style.color = 'green';
+  } else if (60 <= state.tempValue && state.tempValue <= 69) {
+    tempColorElement.style.color = 'yellow';
+  } else if (70 <= state.tempValue && state.tempValue <= 79) {
+    tempColorElement.style.color = 'orange';
+  } else {
+    tempColorElement.style.color = 'red';
+  }
+};
+
+// Function to change landscape of temp value according to temp range
+const landscapeChangeAction = () => {
```

Comment:
Very minor callout - You've been following convention where you name a method with verb+noun like `updateSkyAction`. Here you could do the same to indicate what the method does `changeLandscapeAction`